test/lib.bash

```diff
@@ -95,7 +100,12 @@ function serverless_operator_kafka_e2e_tests {
   done
   kubeconfigs_str="$(array.join , "${kubeconfigs[@]}")"
 
-  go_test_e2e -failfast -tags=e2e -timeout=30m -parallel=1 ./test/e2ekafka \
+  RUN_FLAGS=("-failfast -timeout=30m -parallel=1")
```

Comment:
With this approach, all the test suites now share the same RUN_FLAGS. So, you can't set it differently for individual test suites. I guess this is alright if it works on your side. We don't need to change the flags, we're fine with the defaults.

Reply:
Can you please use just strings instead of arrays in all places? And then use it as "${RUN_FLAGS}" I've put the same comments on the other PR from you team: https://github.com/openshift-knative/eventing/pull/121#discussion_r1107205782